spec/views/users/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "users/edit", type: :view do
```

Comment:
How this is working? I didn't see a route to users/edit.html nor `index`, `show`, `new`.
And why all these features if the PR is about signing up?

Reply:
To start development I used the scaffold generator to create all boilerplate. It generated those features as well and I accidentally committed them.